manim/mobject/vector_field.py

```diff
@@ -208,11 +208,54 @@ def construct(self):
         """
 
         step_size = dt / substeps
+
+        def runge_kutta_pointwise(self, p: Sequence[float], step_size: float) -> float:
+            """Returns the change in position of a point along a vector field.
+            Parameters
+            ----------
+            p
+               The position of each point being moved along the vector field.
+            step_size
+               A scalar that is used to determine how much a point is shifted in a single step.
+
+            Returns
+            -------
+            float
+               How much the point is shifted.
+            """
+            k_1 = self.func(p)
+            k_2 = self.func(p + step_size * (k_1 * 0.5))
+            k_3 = self.func(p + step_size * (k_2 * 0.5))
+            k_4 = self.func(p + step_size * k_3)
+            return step_size / 6.0 * (k_1 + 2.0 * k_2 + 2.0 * k_3 + k_4)
+
+        def runge_kutta_center(self, mob: "Mobject", step_size: float) -> float:
```

Comment:
I'm not sure a second RK function is needed. From the looks of it, they do the same thing, but one acts on `p` and the other on `mob.get_center()`. So why not use the first function, and pass in `mob.get_center()` as an argument?